testing/crosstool-ng/APKBUILD

```diff
@@ -1,13 +1,13 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=crosstool-ng
-pkgver=1.23.0
+pkgver=1.24.0
 pkgrel=1
 pkgdesc="tool for building toolchains"
 url="http://crosstool-ng.org/"
 license="GPL"
```

Comment:
GPL is not spdx and is ambiguous

Reply:
Updated licenses info. Thank you for pointing out the need of this update!